src/main/kotlin/frc/robot/subsystems/swerve/SwerveConstants.kt

```diff
@@ -107,7 +107,9 @@ object SwerveConstants {
     var MAX_OMEGA_VELOCITY = 0.0
     var VY_NOTE_DETECTION_CONTROLLER = PIDController(5.0, 0.0, 0.3)
 
-    val TURN_MAX_TOLERANCE = 3.0/360.0 //TODO: calibrate
+    const val TURN_MAX_TOLERANCE = 3.0/360.0 //TODO: calibrate
+    const val SKID_TOLERANCE = 0.3
+    const val COLLISION_TOLERANCE: Float = 1.8F // [G]
```

Comment:
Use `Measure`s.

Reply:
[G] Units do not exist in the Units class

Reply:
1. [G] is not a unit, it's a constant in acceleration units. [WPILib does have it though](https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/units/Units.html#Gs)
2. What prevents you from using `m/s^2` directly? Or declaring your own `G` measure and reference it?

Reply:
6af35ff00139a56d8a2c8f3e43b544ba2694e1e4